config/crd/bases/datasciencepipelinesapplications.opendatahub.io_datasciencepipelinesapplications.yaml

```diff
@@ -105,6 +105,11 @@ spec:
                       Server. Setting Deploy to false disables operator reconciliation.
                       Default: true'
                     type: boolean
+                  enableIlabKFPPipeline:
+                    default: false
+                    description: 'Include ilab KFP pipeline with the deployment of
```

Comment:
similar to above, lets remove 'KFP' and expand `ilab` to `Instructlab`.  Perhaps include a little more context, like

`Include multi-phase instructlab training pipeline with the deployment of this DSP API Server. Default: false`